jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -21,6 +21,7 @@ pipeline {
             steps {
                 script { 
                     dockerAgent = detectDockerAgent()
+                    currentBuild.description = "$INPUT_MANIFEST"
```

Comment:
I missed this in the CR, it should have probably happened before `detectDockerAgent()`